polaris/curation/viz_utils.py

```diff
@@ -23,27 +33,32 @@ def visualize_distribution(data_cols: List[str], dataset: PandasDataFrame):
     for col in data_cols:
         outlier_col = f"OUTLIER_{col}"
         if outlier_col not in dataset.columns:
-            raise ValueError(
-                f"{outlier_col} is not found in the given dataset.\
-                             Please run <polaris.curation.utils.outlier_detection> first."
+            logger.warning(
+                f"{outlier_col} is not found in the given dataset. Please run <polaris.curation.utils.outlier_detection> first."
             )
```

Comment:
Raising an error here makes more sense to me, because the method otherwise does nothing. Why did you change it?

Reply:
You also throw an error for the `cliff_coll`. I think we should be consistent!